upgrade-instructions.md

```diff
@@ -33,7 +33,7 @@ Concerned products: Guice distributed James (rabbitMQ)
 
 The distributed James project (relying on Guice, Cassandra, ElasticSearch, RabbitMQ and optionally Swift) benefits from a new distributed task mananger.
 
-In order to enforce task sequential processing, we rely on an exclusive consumer, which is a feature introduced in RabbitMQ 3.8.
+In order to enforce task sequential processing, we rely on a single active consumer, which is a feature introduced in RabbitMQ 3.8.
```

Comment:
`In order to enforce task sequential processing via a signle active consumer at the cluster level, we rely on an exclusive consumer, which is a feature introduced in RabbitMQ 3.8`

Reply:
ok I'm doing the update (and a squash because the fixup are not referencing the right commit)

Reply:
in fact not, the naming is the good one: http://next.rabbitmq.com/consumers.html#single-active-consumer
but I will add `at the cluster level` (and do the squash)